website/documentation/appendix/releasenotes/0.10.0-M1.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: documentation
+title: Release Notes
+---
+{% include base.html %}
+## Release Notes - 0.10.0-M1
+
+We are proud to announce the release of 0.10.0-M1.  
+Here are some of the major highlights:
+
+- **Web Editor** that allows to create information, function blocks and entity type models through the web browser
+- New **Code Generators** have been added to the list of existing generators.
+- Bug fixes and improvements
+
+
+### Web Editor
+With the **Web Editor**, you can create individual projects where you can describe the characteristics and functionality of the Information Models and publish then to Vorto Repository for others who could use to generate code using Code Generators. This is solely done using a web browser.
+
+  ![WebEditor]({{base}}/img/documentation/webeditor_details.png){:width="700px"}
+  
+### Code Generators
+
+We have three new Code Generators out of the box, those are
+
+* Bosch IoT Suite
+* Eclipse Kura
+* Web-based Device Dashboard
+
+#### Bosch IOT Suite
+
+This **Bosch IOT Suite** generator allows you to choose and  generate client code for **Device Simulator**, **Alexa Skillset** and **Device Validation** types.
```

Comment:
The Alexa Skill set Generator for Bosch IoT Suite generates an Alexa Skill Interaction model and Lambda function that fetches the device data from Bosch IoT Suite.
The Device Simulator Generator for Bosch IoT Suite generates a simulator that creates test data for a given information model and publishes the data to the Bosch IoT Suite.